packages/components/forms/src/select/Select.styles.ts

```diff
@@ -60,8 +77,8 @@ export function getSelectStyles(props: {
     ),
     select: cx(
       select,
-      props.isDisabled ? disabled : null,
-      props.isInvalid ? invalid : null,
+      isDisabled ? disabled : null,
+      isInvalid ? invalid : null,
```

Comment:
```suggestion
      isDisabled && disabled,
      isInvalid && invalid,
```

I think this will work, maybe you will need to do: 

```
cx([
  select,
  isDisabled && disabled,
  isInvalid && invalid,
])
```